Makefile

```
@@ -83,9 +82,8 @@ clean:
 	rm --force reports/*.log
 	rm --force reports/*.pdf
 	rm --force reports/*.pytxcode
-	rm --force NAMESPACE
 
-coverage: $(jsonLogisticModelParameters)
+coverage: $(jsonLogisticModelParameters) install
```

Intenta cambiando el orden de los requerimientos. Yo esperaría que e; orden no importara, pero tal vez sí.
```suggestion
coverage: install $(jsonLogisticModelParameters)
```